go.mod

```diff
@@ -98,10 +92,189 @@ require (
 	golang.org/x/time v0.0.0-20210220033141-f8bda1e9f3ba
 	google.golang.org/genproto v0.0.0-20210825212027-de86158e7fda
 	google.golang.org/grpc v1.43.0
-	gopkg.in/mgo.v2 v2.0.0-20190816093944-a6b53ec6cb22 // indirect
 	gopkg.in/yaml.v2 v2.4.0
 	upper.io/db.v3 v3.7.1+incompatible
 )
 
+require (
+	cloud.google.com/go v0.93.3 // indirect
+	cloud.google.com/go/storage v1.16.1 // indirect
+	github.com/99designs/keyring v1.1.5 // indirect
```

Comment:
What are these? Can them be removed?

Reply:
it was added by `go mod tidy`, seems cannot be removed